### DISCUSSION TITLE
Grid container

### DISCUSSION THREAD

**Discussion by morningstarsabrina** [zauberzeug/nicegui#344] [Ideas / Feature Requests]
Hello,

I'm trying these days a lot of layout and some times it get tricky(a lot of margin and padding) to do some complex layouts using rows and columns, so I was thinking if its possible to do a grid container (tailwind class) ?
thanks

**Comment by falkoschindler** [comment]
Meanwhile, there is a `ui.grid` element in NiceGUI:

```py
with ui.grid(columns=2):
    ui.label('Name:')
    ui.label('Tom')

    ui.label('Age:')
    ui.label('42')

    ui.label('Height:')
    ui.label('1.80m')
```

https://nicegui.io/documentation/grid
